packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -228,6 +230,44 @@ const Cart = () => {
             setSelectedItem(undefined)
         }
     }
+
+    const handleUpdateBundle = async (bundle, bundleQuantity, childProducts) => {
+        // close the modal before handle the change
+        onClose()
+
+        try {
+            setCartItemLoading(true)
+
+            const item = {
+                productId: bundle.productId,
+                quantity: bundleQuantity,
+                price: bundle.price,
+
+                bundledProductItems: bundle.bundledProductItems.map(({itemId}, i) => {
+                    const quantityPerBundle = bundle.bundledProducts[i].quantity
+                    const totalQuantity = bundleQuantity * quantityPerBundle
+
+                    return {
+                        itemId,
+                        productId: childProducts[i].variant.productId,
+                        quantity: totalQuantity
+                    }
+                })
+            }
+            return await updateItemInBasketMutation.mutateAsync({
```

Comment:
This API operation results strangely in a 500 server error. It used to be ok.. is there something changed on Joel's ODS instance recently? Maybe this is the result of Jainam in the middle of updating his cartridge?

Reply:
Yes. This was me updating the cartridge that was causing 500s